lesson07/assignment/index.html

```diff
@@ -0,0 +1,113 @@
+        <!-- Else commision status is open -->
+        <p class="Main__sidebar__status" v-else>Status: Open</p>
+        <!-- Commission info -->
+        <ul class="Main__sidebar__info" v-if="priceShowing === true">
+          <li class="Main__sidebar__info__price">${{emote}} USD per emote/badge</li>
+          <li class="Main__sidebar__info__price">${{avatar}} USD per avatar/icon</li>
+          <li class="Main__sidebar__info__price">${{panelmin}} - ${{panelmax}} USD per panel (depending on how detailed)</li>
+          <li class="Main__sidebar__info__price">${{half}} USD per half-body</li>
+          <li class="Main__sidebar__info__price">${{full}} USD per full body</li>
+          <li class="Main__sidebar__info__price">+${{bgaddon}} USD per specific background</li>
+          <li class="Main__sidebar__info__price">+${{charaddonmin}} - ${{charaddonmax}} USD per additional character (up to 3 characters per piece, price depends on how detailed)</li>
+        </ul>
+        <!-- Button to toggle the commission prices -->
+        <button class="Main__sidebar__button" type="button" name="button" v-on:click="priceToggle" v-if="priceShowing === true">Hide Prices</button>
+        <button class="Main__sidebar__button" type="button" name="button" v-on:click="priceToggle" v-else>Show Prices</button>
+      </aside>
+      <!-- Page Footer -->
+      <footer class="Main__footer footer">
+        <p class="Main__footer__creation">Originally created from scratch in Spring 2021. Updated regularly.</p>
+      </footer>
+    </main>
+
+    <script type="text/javascript">
+      // Commission info data
+      const CommInfo = {
+        data() {
+          return {
+            // My name
+            name: 'Kendall Eng',
+            // Position title
+            title: 'Creative',
+            // Commission status
+            open: false,
+            // Emote/Badge price
+            emote: 20,
+            // Avatar/Icon price
+            avatar: 30,
+            // Panel price
+            panelmin: 20,
+            panelmax: 40,
+            // Half-body price
+            half: 40,
+            // Full body price
+            full: 50,
+            // Background Add-on price
+            bgaddon: 5,
+            // Character Add-on price
+            charaddonmin: 10,
+            charaddonmax: 20,
+            // Prices showing
+            priceShowing: true,
+            // Array of images
+            imageSrc: ['https://picsum.photos/250', 'https://picsum.photos/250?grayscale', 'https://picsum.photos/250', 'https://picsum.photos/250?grayscale', 'https://picsum.photos/250', 'https://picsum.photos/250?grayscale']
+          }
+        },
+        methods: {
+          // Toggles whether the commission price list is showing
+          priceToggle() {
+            if(this.priceShowing === false) {
+              this.priceShowing = true;
+            } else {
+              this.priceShowing = false;
+            }
```

Comment:
While this totally works, you can also turn this into a one-liner with this:
`this.priceShowing = !this.priceShowing`

The `!` will reverse the bool value in front of it